CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # CHANGELOG
 
+## Version 3.6.2 GA (10/23/2024)
+
+### Enhancements
+
+* Update azure-monitor-opentelemetry-exporter to 1.0.0-beta.31
+  ([#3932](https://github.com/microsoft/ApplicationInsights-Java/pull/3932))
```

Comment:
```suggestion
* Update to OpenTelemetry Java Instrumentation 2.9.0
  ([#3918](https://github.com/microsoft/ApplicationInsights-Java/pull/3918))

### Bug fixes

* Restore missing self-diagnostic log
  ([Azure/azure-sdk-for-java#42451](https://github.com/Azure/azure-sdk-for-java/pull/42451))
```

Reply:
why does it need to repeat the same bug fixes in multiple places?

Reply:
exporter update includes those two. i don't think we need to repeat it